doctr/models/detection/linknet/tensorflow.py

```diff
@@ -20,16 +20,10 @@
 
 from .base import LinkNetPostProcessor, _LinkNet
 
-__all__ = ["LinkNet", "linknet_resnet18", "linknet_resnet34", "linknet_resnet50", "linknet_resnet18_rotation"]
+__all__ = ["LinkNet", "linknet_resnet18", "linknet_resnet34", "linknet_resnet50"]
 
 default_cfgs: Dict[str, Dict[str, Any]] = {
     "linknet_resnet18": {
-        "mean": (0.798, 0.785, 0.772),
-        "std": (0.264, 0.2749, 0.287),
-        "input_shape": (1024, 1024, 3),
-        "url": "https://doctr-static.mindee.com/models?id=v0.6.0/linknet_resnet18-611b50f2.zip&src=0",
```

Comment:
This one should be kept and remove the other one ?

Reply:
Mh it's the same arch so i changed the chekpoint because the sec one was already trained on rotated samples so while finetuning it should converge faster as the checkpoint which wasn't trained with rotated samples :)

Makes only sense for TF because the change to antialias=True was much less changing as compared to pytorch

Reply:
In the end we'll train it anyway ^^